tests/python/pants_test/bin/test_runner_integration.py

```diff
@@ -37,3 +38,41 @@ def test_warning_filter(self):
     })
     self.assert_success(non_warning_run)
     self.assertNotIn('test warning', non_warning_run.stderr_data)
+
+  def test_parent_build_id_set_only_for_pants_runs_called_by_other_pants_runs(self):
+    with self.temporary_workdir() as workdir:
+      config = {
+        'GLOBAL': {
+          'colors': True,
```

Comment:
Why do we specifically enable `colors` for this test?

Reply:
testprojects/src/python/nested_runs require to have a config. This is the way I choose to create a config that doesn't anything useful but satisfies the requirement to have a config file.